src/calculateRentalCost.js

```diff
@@ -5,6 +5,15 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  if (days >= 7) {
```

Comment:
Replace the literals '7' and '50' with constants to represent the long-term rental threshold and discount, respectively. This will make your code more maintainable and align with checklist item 2.